src/main/scala/com.socrata.tileserver/util/GeoProvider.scala

```diff
@@ -25,8 +25,6 @@ case class GeoProvider(client: CuratedServiceClient) {
   def doQuery(info: RequestInfo): GeoResponse = {
     val intersects = filter(info.tile, info.geoColumn, info.overscan.getOrElse(0))
     val params = augmentParams(info, intersects)
-    val paramsWithTimeout = addQueryTimeout(params, config.TileServerConfig.queryTimeout)
```

Comment:
We don't need `config.TileServerConfig.queryTimeout` anymore in our request? Or is it getting in there some other way?